src/core/tests/Test_PatchInstaller.py

```diff
@@ -270,11 +270,11 @@ def test_stopwatch_properties_patch_install_fail(self):
     def test_write_installer_perf_logs_catch_exception(self):
         # Testing the catch Exception in the method write_installer_perf_logs
         # ZeroDivisionError Exception should be thrown by the function get_percentage_maintenance_window_used because denominator will be zero if maximum_duration is zero
-        # This will cover the catch exception code but another exception will be thrown later due to calling stop_and_write_telemetry without initializing stopwatch
+        # This will cover the catch exception code
         argument_composer = ArgumentComposer()
         argument_composer.maximum_duration = "PT0H"
         runtime = RuntimeCompositor(argument_composer.get_composed_arguments(), legacy_mode=True)
-        self.assertRaises(Exception, runtime.patch_installer.write_installer_perf_logs, True, 1, 1, runtime.maintenance_window, False, Constants.TaskStatus.SUCCEEDED, "")
+        runtime.patch_installer.write_installer_perf_logs(True, 1, 1, runtime.maintenance_window, False, Constants.TaskStatus.SUCCEEDED, "")
```

Comment:
What exactly are you testing here? This needs some checks.

Reply:
There is a try catch block in write_installer_perf_logs method. This test case covers the catch block of the try catch block.

ZeroDivisionError Exception should be thrown by the function get_percentage_maintenance_window_used because denominator will be zero if maximum_duration is zero. This test is for code coverage of catch block.

write_installer_perf_logs method does not return anything. It does add log in telemetry. So, there are no checks.

Reply:
Can we add a return True in that method (write_installer_perf_logs ) and test for the same here. This return will only be used in the test.

Reply:
Done